docs/docs/api/contacts.md

```diff
@@ -162,45 +229,6 @@ All fields set on the [creation request](#create-contact) can be updated. To rem
 
 **Sample response 202 (Accepted)**
 
-### Re-fetch contact
-
-`POST /contacts.refetch`
-
-If there are conversations associated with this contact the sources will try to re-fetch the contact data that was ingested. This only works for messaging sources that allow you to pull contact data such as [Facebook Messenger](https://developers.facebook.com/docs/messenger-platform/identity/user-profile/). Others like Google Business Messages will push new contact information, which will cause it to show up automatically.
-
-**Sample request**
-
-```json5
-{
-  "contacts": ["6b80b10c-ae6e-4995-844d-c56c4da11623", "c564cea4-a96f-4ebb-a220-3fb81b6ad522"]
-}
-```
-
-**(202) Sample response**
-
-### Merge contacts
```

Comment:
Merge does not behave this way anymore. The documentation will be updated when `merge` is implemented